### ISSUE TITLE
melons3 firmware problem with qwavefpga.h

### ISSUE THREAD

**Issue by imangholizadeh** [QWaveSystems/Melon_S3_FPGA#3]
I program my esp8266 board (ESP8266 ESP-12E NODEMCU ) with Firmware-MelonS3-v.0.3.ino without any change and  then i cant see any access point. i debuged code and i saw  functions that realated to qwavefpga.h  not runing. why?
i dont have wroom-2 and i want to use ESP8266 ESP-12E NODEMCU instad of  wroom-2.can i do this? how?

**Comment by pamornthep** [comment]
The code "Firmware-MelonS3-v.0.3.ino " haven't tested with ESP-12E. I am not sure what is the problem. I only do tested with WROOM-02, The WROOM-02 module is difference from ESP-12E in term of hardware spec,flash memory,pin assignment and etc...

**Comment by imangholizadeh** [comment]
thanks for your replay. I know WROOM-02 module is difference from ESP-12E and i change spec pin configuration and etc ... but no answer. my problem is There are no WROOM-02 in my country, and if they exist, they are much more expensive than ESP-12E. if i has qwavefpga.ccp i can write this firmware for other arduino board but in melon s3 project qwavefpga.ccp is hidden in static library "libqwavesys.a " located in "\qwavesys\hardware\esp8266\2.3.0-2\tools\sdk\lib" and i can not access to source of "libqwavesys.a " . if melon s3 project is open source  I expected that is not hidden. If you can help me anyway, I will thank you for it.

**Comment by johnmayer21** [comment]
@imangholizadeh They can not tell you this because they are just copying our work and they are not aware of it themselves.
This science is only available to our company, and they will always be illegal in our area.

**Comment by johnmayer21** [comment]
@imangholizadeh They can not tell you this because they are just copying our work and they are not aware of it themselves.
This science is only available to our company and they will always follow us.

**Comment by sdikiy** [comment]
Very strange situation.
The project claims GPL support, and in fact most of the code is hidden in a static library.
https://github.com/QWaveSystems/Melon_S3_FPGA/blob/master/Firmware/License.md
Very strange project ...

**Comment by imangholizadeh** [comment]
Yes Very strange project ...and we can not use that because all required code is hidden in a static library.
if you want to share some information with together you can send me email. 
Email address: imangholizadeh68@gmail.com
